aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+
+.. code-block:: Json 
+
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "1.0.0-*",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
+    }
+  },
+  "scripts": {
+    "postpublish": "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%"
+  }
+
+The ASP.NET Core Module must be configured in ``web.config``:
+
+.. code-block:: Xml
+  
+  <configuration>
+    <system.webServer>
+      <handlers>
+        <add name="aspNetCore" path="*" verb="*" 
+		modules="AspNetCoreModule" resourceType="Unspecified"/>
+      </handlers>
+      <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" 
+	  stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" 
+	  forwardWindowsAuthToken="false"/>
+    </system.webServer>
+  </configuration>
+
+
+The Publish IIS tool 
+^^^^^^^^^^^^^^^^^^^^^
+
+The name of the package that contains the Publish IIS tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your project.json file to inlude the ``Microsoft.AspNetCore.Server.IISIntegration.Tools`` package instead of the ``dotnet-publish-iis`` package.
+
+You can use the Publish IIS tool to publish your app with the ``web.config`` file that is required for your target environment. More information about Publish IIS changes is at https://github.com/aspnet/Announcements/issues/164.
+
+IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main``, and is no longer called in the ``Configure`` method of the ``Startup`` class.
+
+Json configuration syntax change 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``ConfigurationRoot.ReloadOnChanged`` is no longer available, add is added explicitly via ``ConfigurationBuilder.AddJsonFile``. Update ``launchSettings.json`` to remove web target and add the following:
+
+.. code-block:: Json 
+
+    "WebApplication1": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "http://localhost:5000",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+
+
+Changes in MVC
+--------------
+
+To compile views, set the ``preserveCompilationContext`` option in ``project.json`` to preserve the compilation context, as shown here:
+
+.. code-block:: c#  
+
+  {
+    "userSecretsId": "aspnet-WebApplication11-b15b61be-47f8-499c-ac42-9f74b8cdbebe",
+    "compilationOptions": {
+      "emitEntryPoint": true,
+      "preserveCompilationContext": true
+  },
+
+
+You no longer need to reference the Tag Helper package ``Microsoft.AspNet.Mvc.TagHelpers``, which was renamed to ``Microsoft.AspNetCore.Mvc.TagHelpers`` in RC2. The package is now referenced by MVC by default.
+
+Changes in views
+^^^^^^^^^^^^^^^^
+
+Views now support relative paths. 
+
+The Validation Summary Tag Helper has changed. 
+
+RC1:
+
+.. code-block:: html 
+
+  <div asp-validation-summary="All" class="text-danger"></div> 
```

Comment:
I think you have this backwards, RC2 is this syntax and the one below is the RC1 syntax